test/libsolidity/syntaxTests/literalSuffixes/parameters/overloading_decimal_vs_decimal.sol

```diff
@@ -0,0 +1,21 @@
+function uSuffix(uint8, uint) pure suffix returns (uint) {}
+function uSuffix(uint16, uint) pure suffix returns (int) {}
+
+function iSuffix(int8, uint) pure suffix returns (uint) {}
+function iSuffix(int16, uint) pure suffix returns (int) {}
+
+function iuSuffix(uint8, uint) pure suffix returns (int) {}
+function iuSuffix(int8, uint) pure suffix returns (uint) {}
+
+contract C {
+    int a = 1.024 uSuffix;
+    int b = 1.024 iSuffix;
+
+    int c = -1.024 uSuffix;
+    int d = -1.024 iSuffix;
+
+    int e = 2.55 iuSuffix;
+    int f = -2.55 iuSuffix;
+}
+// ----
+// TypeError 2144: (392-399): No matching declaration found after variable lookup.
```

Comment:
It's impossible to judge which statement is triggering this error. Should all cases here be invalid, but you're only actually testing one?

Reply:
There's similar issue with quite a number of other tests when I'm skimming through them.

Reply:
It's because they were all valid back when overloading was allowed.

Reply:
Yes, I know, but do we really want to keep them like that?

Reply:
I removed lines that are unreachable after a fatal error from some test cases and split some others. I hope it's better now.